src/main/java/io/jenkins/plugins/CheckGHEventSubscriber.java

```diff
@@ -59,16 +60,41 @@ protected boolean isApplicable(@Nullable Item project) {
     @Override
     protected void onEvent(GHSubscriberEvent event) {
         switch (event.getGHEvent()) {
-            case CHECK_SUITE:
-                LOGGER.log(Level.FINE, "Received Check Run Event...");
-                checkSuiteEvents.add(event);
-                break;
-            case CHECK_RUN:
-                LOGGER.log(Level.FINE, "Received Check Suite Event...");
-                checkRunEvents.add(event);
+            case INSTALLATION_REPOSITORIES:
+                LOGGER.log(Level.FINE, "Received Installation Repositories Event...");
+                updateInstallations(event); // add (repository_name, installation_id) pair
                 break;
             default:
                 throw new IllegalStateException("Unexpected value: " + event);
         }
     }
+
+    /**
+     * Update installations
+     *
+     * @param event installation repositories event
+     */
+    private void updateInstallations(GHSubscriberEvent event) {
+        JsonNode payload = null;
+        try {
+            payload = new ObjectMapper().readTree(event.getPayload());
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        if (payload == null)
+            return;
+
+        long installationId = payload.get("installation").get("id").asLong();
+
+        // iterate the added repositories
+        Iterator<JsonNode> repoIter = payload.get("repositories_added").iterator();
+        while (repoIter.hasNext())
+            repoFullNameToInstallationId.put(repoIter.next().get("full_name").asText(), installationId);
+
```

Comment:
formatting might not be right but please use braces so that if someone adds a new line it's clear if its in the loop or not
```suggestion
        while (repoIter.hasNext()) {
            repoFullNameToInstallationId.put(repoIter.next().get("full_name").asText(), installationId);
            }

```